<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QProgressBar" name="prog_top">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>875</width>
      <height>5</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="prog_bottom">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>635</y>
      <width>875</width>
      <height>5</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="invertedAppearance">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="prog_right">
    <property name="geometry">
     <rect>
      <x>885</x>
      <y>10</y>
      <width>5</width>
      <height>625</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="invertedAppearance">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="prog_left">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>5</width>
      <height>625</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
